sse2neon.h

```diff
@@ -7592,15 +7592,8 @@ FORCE_INLINE int _mm_test_all_zeros(__m128i a, __m128i mask)
 // zero, otherwise set CF to 0. Return 1 if both the ZF and CF values are zero,
 // otherwise return 0.
 // https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=mm_test_mix_ones_zero
-FORCE_INLINE int _mm_test_mix_ones_zeros(__m128i a, __m128i mask)
-{
-    uint64x2_t zf =
-        vandq_u64(vreinterpretq_u64_m128i(mask), vreinterpretq_u64_m128i(a));
-    uint64x2_t cf =
-        vbicq_u64(vreinterpretq_u64_m128i(mask), vreinterpretq_u64_m128i(a));
-    uint64x2_t result = vandq_u64(zf, cf);
```

Comment:
I think fixed this line with the following code can solve the issue
```diff
-uint64x2_t result = vandq_u64(zf, cf);
+uint64x2_t result = vreinterpretq_u64_u32(vmvnq_u32(vreinterpretq_u32_u64(vorrq_u64(zf, cf))));
```

Reply:
This suffers from the same problem. We can't `OR`/`AND` zf and cf together, until we've normalized then somehow. I'll see what I can do.

Reply:
something like this for ARMv7-A ?
```
FORCE_INLINE int _mm_test_mix_ones_zeros(__m128i a, __m128i mask)
{
    uint64x2_t v = vreinterpretq_u64_m128i(a);
    uint64x2_t m = vreinterpretq_u64_m128i(mask);
    uint32x2_t has_one = vqmovn_u64(vandq_u64(m, v));
    uint32x2_t has_zero = vqmovn_u64(vbicq_u64(m, v));
    uint32x2_t pack = vpmax_u32(has_one, has_zero);
    uint32x2_t reduced = vpmin_u32(pack, pack);
    return (vget_lane_u32(reduced, 0) != 0);
}
```

Reply:
added. remeber to squash it.